build.gradle

```diff
@@ -34,13 +34,17 @@ dependencies {
     // or Jenkins plugin versions in Jenkins plugins that depend on this library.
     // Jenkins plugins that depend on this library should therefore version those dependencies compatibly with the versions below:
     // --rotte DEC 2019
-    implementation 'org.jenkins-ci.main:jenkins-core:2.248'
-    implementation 'org.jenkins-ci.plugins:credentials:2.3.12@jar'
-    implementation 'org.jenkins-ci.plugins:credentials:2.3.12'
-    implementation 'org.jenkins-ci.plugins:plain-credentials:1.7@jar'
-    implementation 'org.jenkins-ci.plugins:plain-credentials:1.7'
-    implementation 'org.jenkins-ci.plugins.workflow:workflow-support:3.5@jar'
-    implementation 'org.jenkins-ci.plugins.workflow:workflow-support:3.5'
-    implementation 'org.jenkins-ci.plugins.workflow:workflow-api:2.40@jar'
-    implementation 'org.jenkins-ci.plugins.workflow:workflow-api:2.40'
+
+    // core version bump
+    implementation 'org.jenkins-ci.main:jenkins-core:2.334'
+
+    // the following versions are compatible/usable in jenkins core 2.334 and later apparently.
+    implementation 'org.jenkins-ci.plugins:credentials:1087.1089.v2f1b_9a_b_040e4@jar'
+    implementation 'org.jenkins-ci.plugins:credentials:1087.1089.v2f1b_9a_b_040'
+    implementation 'org.jenkins-ci.plugins:plain-credentials:139.ved2b_9cf7587b@jar'
+    implementation 'org.jenkins-ci.plugins:plain-credentials:139.ved2b_9cf7587b'
+    implementation 'org.jenkins-ci.plugins.workflow:workflow-support:815.vd60466279fc8@jar'
+    implementation 'org.jenkins-ci.plugins.workflow:workflow-support:815.vd60466279fc8'
+    implementation 'org.jenkins-ci.plugins.workflow:workflow-api:1143.v2d42f1e9dea_5@jar'
+    implementation 'org.jenkins-ci.plugins.workflow:workflow-api:1143.v2d42f1e9dea_5'
```

Comment:
All of the Jenkins dependencies updated here were released after jenkins-core:2.334 was released (2/8/22).

Any concerns with dependent plugins being release after jenkins-core:2.334?
Using `org.jenkins-ci.plugins:credentials:1087.1089.v2f1b_9a_b_040` as an example, does this set a minimum version required for the credentials plugin?

Reply:
re: credentials:1087.1089...   I had initially "asked for" a version even more recent than that, when attempting to install into a 2.334 based docker container jenkins.  The message I got back specified this version  was "already there" and I would need to upgrade to that earlier one.  However, that said....

jenkins-core 2.334 was apparently released on Feb 08, 2022

According to the mvn repo pages....
workflow-api 1143...  was released Mar 10, 2022 (https://mvnrepository.com/artifact/org.jenkins-ci.plugins.workflow/workflow-api/1143.v2d42f1e9dea_5)

workflow-support 815...  was released on Mar 2 2022 (https://mvnrepository.com/artifact/org.jenkins-ci.plugins.workflow/workflow-support/815.vd60466279fc8)

The cred plugins in question were released later but they appear to function without warning or issue in the 2.334 based jenkins container.